junit-platform-commons/src/main/java/org/junit/platform/commons/support/ModifierSupport.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2015-2018 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.platform.commons.support;
+
+import static org.apiguardian.api.API.Status.MAINTAINED;
+
+import java.lang.reflect.Member;
+
+import org.apiguardian.api.API;
+import org.junit.platform.commons.util.Preconditions;
+import org.junit.platform.commons.util.ReflectionUtils;
+
+/**
+ * Common modifier support.
+ *
+ * @since 1.4
+ * @see java.lang.reflect.Modifier
+ */
+@API(status = MAINTAINED, since = "1.4.0")
+public final class ModifierSupport {
+
+	/**
+	 * Returns {@code true} if the given class is declared {@code public}, {@code false} otherwise.
```

Comment:
Please update the Javadoc for all methods in this class according to the following guidelines.

- Instead of "Returns ...", we say `Determine if the supplied class is {@code public}.` and rely on the `@return` tag to state what the method returns.
- We typically do not (never?) state things such as "false otherwise" since that is implicit.
- Please change the return tag to `@return {@code true} if the class is {@code public}`.